common/djangoapps/student/urls.py

```diff
@@ -0,0 +1,39 @@
+"""
+URLs for student app
+"""
+
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from student.views import LogoutView
+
+urlpatterns = (
+    'student.views',
+
+    url(r'^dashboard$', 'dashboard', name="dashboard"),
+
+    url(r'^login_ajax$', 'login_user', name="login"),
+    url(r'^login_ajax/(?P<error>[^/]*)$', 'login_user'),
+
+    url(r'^email_confirm/(?P<key>[^/]*)$', 'confirm_email_change'),
+
+    url(r'^accounts/manage_user_standing', 'manage_user_standing', name='manage_user_standing'),
+    url(r'^accounts/disable_account_ajax$', 'disable_account_ajax', name="disable_account_ajax"),
+
+    url(r'^logout$', LogoutView.as_view(), name='logout'),
+    url(r'^create_account$', 'create_account', name='create_account'),
+    url(r'^activate/(?P<key>[^/]*)$', 'activate_account', name="activate"),
+
+    url(r'^change_setting$', 'change_setting', name='change_setting'),
+
+    url(r'^change_enrollment$', 'change_enrollment', name='change_enrollment'),
+    url(r'^change_email_settings$', 'change_email_settings', name='change_email_settings'),
```

Comment:
Is there a more logical ordering of the URLs?
For example, can login and logout URLs be grouped together?  And account creation/management ones together?